rmgpy/molecule/resonance.py

```diff
@@ -482,14 +484,52 @@ def generateAromaticResonanceStructures(mol, features=None):
     for mol0, aromaticBonds in molList:
         if not aromaticBonds:
             continue
+        # Save original bond orders in case this doesn't work out
+        originalBonds = []
+        for ring in aromaticBonds:
+            originalOrder = []
+            for bond in ring:
+                originalOrder.append(bond.order)
+            originalBonds.append(originalOrder)
+        # Change bond types to benzene bonds for all aromatic rings
         for ring in aromaticBonds:
             for bond in ring:
                 bond.order = 1.5
 
         try:
             mol0.updateAtomTypes(logSpecies=False)
         except AtomTypeError:
-            continue
+            # If this didn't work the first time, then there might be a ring that is not actually aromatic
```

Comment:
Do you know if there is a robust way to test this exception for a structure that rdkit always fails on? Are we already testing this somewhere? I didn't notice it explicitly in other commits?